env0/provider.go

```diff
@@ -38,6 +39,7 @@ func Provider(version string) plugin.ProviderFunc {
 			DataSourcesMap: map[string]*schema.Resource{
 				"env0_organization":           dataOrganization(),
 				"env0_project":                dataProject(),
+				"env0_policy":                 dataPolicy(),
```

Comment:
This is some sort of general comment about naming of the data/resource entities.
This entity should represent the policies defined under a project. We do have organization policies but it's not so robust nor represented yet by this TF provider.

Due to the above - I'd suggest to rename this from `env0_policy` to `env0_project_policy` (and also the relevant files...)